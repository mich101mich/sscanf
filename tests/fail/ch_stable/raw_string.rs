fn main() {
    sscanf::scanf!("hi", r#"asdf{}asdf"#);
}
