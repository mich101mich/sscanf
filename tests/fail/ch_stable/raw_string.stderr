error: Missing Type for given '{}' Placeholder:
At r#"asdf{}asdf"#
          ^^
 --> $DIR/raw_string.rs:2:26
  |
2 |     sscanf::scanf!("hi", r#"asdf{}asdf"#);
  |                          ^^^^^^^^^^^^^^^
